index.qmd

```diff
@@ -84,48 +84,102 @@ accessibility tests, and more, consult our [Guides](#guides)
 
 ### Installation
 
-To install the workbench, make sure you have a working version of R and 
-pandoc/RStudio installed (see [the workbench setup instructions for 
+:::: {.panel-tabset}
+
+#### Within R
+
+To install the workbench, make sure you have a working version of R and
+pandoc/RStudio installed (see [the workbench setup instructions for
 details](https://carpentries.github.io/sandpaper-docs/)).
 
-::: {.callout-tip collapse='true'}
+From there, you can install the workbench packages and their dependencies from
+our [Carpentries R Universe](https://carpentries.r-universe.dev) inside of R:
+
+```r
+install.packages(c("sandpaper", "varnish", "pegboard", "tinkr"),
+  repos = c("https://carpentries.r-universe.dev/", getOption("repos")))
+```
+
+#### From conda-forge
+
+All of The Workbench tools, and their dependencies, are packaged and distributed
+as built binaries in the form of conda packages on [conda-forge](https://conda-forge.org/)
+for Linux, macOS, and Windows.
+You can setup an environment for lesson development that includes all dependencies
+(including R) with the following.
 
-#### Setup with Anaconda
+::: {.panel-tabset}
+
+##### With Pixi
 
-Thanks to Travis Wrightsman for [providing the instructions to setup for
-anaconda](https://github.com/carpentries/workbench/issues/50). These lines
-will set up an anaconda environment that includes all the dependencies you need
-to use The Workbench. 
+[Pixi](https://pixi.sh/) environments are fully reproducible by default.
+From the project directory, initialize a Pixi workspace
 
 ```sh
-conda create -n workbench 'git>=2.28' 'r-base>=3.6' 'pandoc>=2.11' pkg-config libxslt
-conda activate workbench
-R -e 'install.packages(c("sandpaper", "varnish", "pegboard", "tinkr"), \
-  repos = list(carpentries="https://carpentries.r-universe.dev/", CRAN="https://cloud.r-project.org"))'
+pixi init
 ```
 
-:::
+and then add Git and The Workbench tool packages (`r-pegboard` is a dependency of `r-sandpaper`)
 
-From there, you can install the workbench packages and their dependencies from
-our [Carpentries R Universe](https://carpentries.r-universe.dev) inside of R:
+```sh
+pixi add git r-varnish r-sandpaper
+```
 
-```r
-install.packages(c("sandpaper", "varnish", "pegboard", "tinkr"),
-  repos = c("https://carpentries.r-universe.dev/", getOption("repos")))
+You can optionally activate a shell with the environment loaded with
+
+```sh
+pixi shell
+```
+
+##### With Conda
+
+Create a [conda](https://docs.conda.io/) environment and install Git and The
+Workbench tool packages (`r-pegboard` is a dependency of `r-sandpaper`) from
+only the `conda-forge` channel
+
+```sh
+conda create --name workbench
+conda config --name workbench --add channels conda-forge
+conda config --name workbench --remove channels defaults
+conda install --name workbench git r-varnish r-sandpaper
```

Comment:
```suggestion
conda create --name workbench
conda install --name workbench --override-channels --channel conda-forge git r-tinkr r-varnish r-pegboard r-sandpaper
```

This should have the same effect of disabling the `defaults` channel. I also added the other packages.

Reply:
@ashwinvis Thanks, but no, it doesn't do the same thing. Most people do not want to be using the `defaults` channel **at all** and so this is why I am explicitly removing the channel from the environment so that no future install commands can use it either. Your suggested command just overrides the channel list _for that particular install command only_.

Reply:
I thought the intention was to install sandpaper and nothing more. If people actually adds more packages, it might make sense to do your way